src/main/java/com/yahoo/bullet/parsing/Query.java

```diff
@@ -27,9 +29,9 @@
 @Getter @Setter @Slf4j
 public class Query implements Configurable, Initializable {
     @Expose
-    private Projection projection;
+    private Map<String, LazyExpression> projection;
```

Comment:
If we're inverting this map (field -> newName) => (expression -> name), then let's break the API and make projection => projections and let it be a list of this Projection object. That way we get an ordering too

Reply:
Also, why do we only initialize a Projection object in RunningQuery#initialize? Why don't we parse it right here using GSON?

Reply:
Ok ordering is not as important because we can't easily produce output in the same order (JSON). We should still use parsing.Projection here though.

Reply:
I could make it a List<Projection> projections; and in initialize(), check that all the names are unique. If you want a map in the projection, we could do some interesting things.

"projection": {
    "fields": {
    },
    "post": {
    }
}